modules/download_lightsail_ssh_keys/main.py

```diff
@@ -34,29 +33,47 @@
 
 def main(args, pacu_main):
     ###### Don't modify these. They can be removed if you are not using the function.
+    session = pacu_main.get_active_session()
     args = parser.parse_args(args)
     print = pacu_main.print
     get_regions = pacu_main.get_regions
     ######
     summary_data = {'region_key_pairs': []}
     regions = get_regions('lightsail')
 
+    dl_path = Path.cwd() / 'sessions' / session.name / 'downloads' /'download_lightsail_ssh_keys'
+    if not dl_path.exists():
+        dl_path.mkdir()
+    summary_data['dl_path'] = str(dl_path.relative_to(Path.cwd() / 'sessions' / session.name))
     for region in regions:
+        print('  Downloading default keys for {}...'.format(region))
+        cur_path = dl_path / region
+        if not cur_path.exists():
+            cur_path.mkdir()
         client = pacu_main.get_boto3_client('lightsail', region)
         downloaded_keys = client.download_default_key_pair()
         restructured_keys = {
             'publicKey': downloaded_keys['publicKeyBase64'],
             'privateKey': downloaded_keys['privateKeyBase64']
         }
-        print('Region: {}\n{}\n'.format(region, json.dumps(restructured_keys)))
+
+        private_path = cur_path / 'default'
+        with private_path.open('w', encoding='utf-8') as key_file:
```

Comment:
I set the flag to 'w' because I just wanted to write the files. I didn't want there to be a chance that the old data persisted in that key-file because the latest key downloaded should be the most correct.